core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -3126,7 +3126,8 @@ exprList: expr (COMMA expr)*
     ;
 
 expr
-    : L_PAREN expr R_PAREN                      # exprPrecedence
+    :
+    L_PAREN expr R_PAREN                      # exprPrecedence
```

Comment:
why this line was changed?

Reply:
will take care of it. Its work in progress and a draft PR, As a practice I commit the dev code on regular basis.